### PR TITLE
readme: clarify ROS msg support early on

### PR DESCRIPTION
**Pull request by gavanderhoorn**
As per subject.

A little bit of duplication, but I believe it's worth it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,22 @@ The mod itself is hosted in this repository.
 [tud-cor/fs_mod_ros](https://github.com/tud-cor/fs_mod_ros) contains two example ROS packages which show how to interact with FarmSim19 running `modROS`.
 
 
+### ROS message support
+
+In its current state, `modROS` publishes four types of messages:
+
+ - `rosgraph_msgs/Clock`: in-game simulated clock. This message stops being published when the game is paused/exited
+ - `nav_msgs/Odometry`: ground-truth `Pose` and `Twist` of vehicles based on their in-game position and orientation
+ - `sensor_msgs/LaserScan`: from five laser scanners sharing a common origin at the front of the vehicle
```
Comment (tckarenchiang) [inline README.md:18]: If we merge #20, then it's not "a common origin" anymore

Comment (gavanderhoorn) [reply README.md:18]: True.

But let's update the docs if/when #20 gets merged.

Comment (gavanderhoorn) [reply README.md]: Updated.

**Comment by gavanderhoorn** [comment]
Rebased and fixed the laser scanner description.

**Review by tckarenchiang — APPROVED** [review]

**Pull request merged**